goldstone/core/tests_resource_api_1.py

```diff
@@ -357,7 +357,9 @@ def test_mix(self):
         EXPECTED.sort()
         content = json.loads(response.content)["nodes"]
         content.sort()
-        self.assertEqual(content, EXPECTED)
+
+        for item in EXPECTED:
+            self.assertTrue(item in content)
 
```

Comment:
The reason for the changes was to make the tests applicable when addons are installed.  In that case, the lists are not equal, so the test checks that all of the expected members of the list are there.  I tried assertIn, but was unsuccessful.